src/index.html

```diff
@@ -1,13 +1,256 @@
+            <p class="advantages__text">The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.</p>
+          </article>
+
+          <article class="advantages">
+            <a href="#" class="advantages__link advantages__link--wide">
+              <img
+                class="advantages__img"
+                src="./images/advantage-range.png"
+                alt="km"
+              >
+            </a>
+
+            <a href="#" class="advantages__link advantages__link--square">
+              <img
+                class="advantages__img"
+                src="./images/advantages-lights.png"
+                alt="km"
+              >
+            </a>
+            <h4 class="advantages__title">Range & Integrated lights</h4>
+            <p class="advantages__text">The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+          </article>
+
+          <article class="advantages">
+            <a href="#" class="advantages__link advantages__link--square">
+              <img
+                class="advantages__img"
+                src="./images/advantages-brake.png"
+                alt="km"
+              >
+            </a>
+
+            <a href="#" class="advantages__link advantages__link--wide">
+              <img
+                class="advantages__img"
+                src="./images/advantages-disk.png"
+                alt="km"
+              >
+            </a>
+            <h4 class="advantages__title">Hydraulic disc brakes & Lightweight</h4>
+            <p class="advantages__text">Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+          </article>
+        </div>
+        <button class="details__btn">Explore</button>
+      </section>
+
+      <section id="contacts"  class="contacts">
+        <h2 class="contacts__title">Contact us</h2>
+        <form action="info" class="contacts__form">
+          <label for="name" class="contacts__label">
+            <input
+              name="name"
+              type="text"
+              placeholder="Name"
+              class="contacts__input"
+            >
+          </label>
+
+          <label for="email" class="contacts__label">
+            <input
+              name="email"
+              type="email"
+              placeholder="Email"
+              class="contacts__input"
+            >
+          </label>
+
+          <label for="text" class="contacts__label contacts__label--message">
+            <textarea
+              type="text"
+              name="text"
+              id="contacts__text"
+              class="contacts__textarea"
+              placeholder="Message"
+            >
+            </textarea>
```

Comment:
```suggestion
              placeholder="Message"
            ></textarea>
```

Update format to fix extra spaces in textarea field